datafusion/expr-common/src/signature.rs

```diff
@@ -125,6 +125,11 @@ pub enum TypeSignature {
     /// Fixed number of arguments of numeric types.
     /// See <https://docs.rs/arrow/latest/arrow/datatypes/enum.DataType.html#method.is_numeric> to know which type is considered numeric
     Numeric(usize),
+    /// Fixed number of arguments of string types.
```

Comment:
> Fixed number of arguments of string types.

Currently implementation can be better documented as "Fixed number of arguments of all of same string type (all of Utf8 type, or all of LargeUtf8 type or all of Utf8View type)".

If this is indeed semantics we want (see comment in `datafusion/expr/src/type_coercion/functions.rs` for that), let's make sure the documentation makes it clear.